src/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+option(BUILD_STATIC "Build static lib" ON)
+option(BUILD_SHARED "Build shared lib" ON)
+
+file(GLOB SRC_FILES
+    ${PROJECT_SOURCE_DIR}/src/**/**/**/*.cpp
+    ${PROJECT_SOURCE_DIR}/src/**/**/*.cpp
+    ${PROJECT_SOURCE_DIR}/src/**/*.cpp
+    ${PROJECT_SOURCE_DIR}/src/*.cpp
+)
+
+add_compile_options(
```

Comment:
I think that glob is a bad idea for CMakelists.txt

Mostly because if @Damdoshi Want to target Windows (MSVC).

We will need to optional exclude some of the source file (opencv, libusb, stuff like that) 

I'm more about adding bunch of options in the CMakeLists.txt

`BUNNY_WITH_OPENCV` etc stuff like that

Reply:
I fully agree, glob were just used to quickly get a working example